src/templates/order/admin/refund/content/sections/form/table.html.twig

```diff
@@ -0,0 +1,10 @@
+<table id="refunds" class="ui compact celled table">
+    <thead>
+    <tr>
+        {% hook 'table.header' %}
+    </tr>
+    </thead>
+    <tbody>
+        {% hook 'table.body' %}
+    </tbody>
```

Comment:
It could be confusing not to have a `table` hook, I don't know if it wouldn't be better to add an additional template with it and trigger these two there 🤔